scripts/build_android.sh

```diff
@@ -52,24 +52,35 @@ CMAKE_ARGS=()
 # target architecture and need an exact version match.
 CMAKE_ARGS+=("-DCAFFE2_CUSTOM_PROTOC_EXECUTABLE=$CAFFE2_ROOT/build_host_protoc/bin/protoc")
 
+CMAKE_ARGS+=($@)
+USE_ARM_COMPUTE=false
+for arg in ${CMAKE_ARGS[@]};do
+  if [ $arg == "-DUSE_ARM_COMPUTE=ON" ];then
```

Comment:
there are a couple other ways to set things "true" in cmake, it might be good to check for all of those

Reply:
-DUSE_ARM_COMPUTE=1? what else?